lib/utils/forkedblockchain.js

```diff
@@ -234,52 +216,46 @@ ForkedBlockchain.prototype.getFallbackBlock = function(numberOrHash, cb) {
 };
 
 ForkedBlockchain.prototype.getBlock = function(number, callback) {
-  var self = this;
-
-  this.isFallbackBlockHash(number, function(err, isFallbackBlockHash) {
+  let checkFn;
```

Comment:
Reason for change: simplifying some of the code, which also eliminates from duplicate checks.